### ISSUE TITLE
PublishToOpsServer.py : Failed to execute UploadServiceDefinition

### ISSUE THREAD

**Issue by jerrym26** [Esri/ops-server-config#741]
I try to use the PublishToOpsServer.py but none of the service is published

![pbscript_publi](https://cloud.githubusercontent.com/assets/11562850/6741633/852e9294-ce89-11e4-8af6-0f43e28f0187.png)

note : try to publish thru the manager a servie and the service was created > so the server is responding "well"
Jérémie

**Comment by jerrym26** [comment]
Update  / Another try another message : 

---------------------------------------------------------------------------
- Registering temporary 'replicated' set of data stores to use during publishing...
---------------------------------------------------------------------------

[u'Failed to log in. Invalid username or password specified.']

***** ERROR ENCOUNTERED *****
PYTHON ERRORS:
Traceback info:
  File "C:\ops-server-config\Publish\Server\PublishToOpsServer.py", line 372, in <module>
    success, dataStorePaths = registerDataStores()

Error Info:
None


***** ERROR: Publishing services to ArcGIS Server site was _NOT_ completed successfully.

Start time: 2015-03-19 23:10:16.973000
End time: 2015-03-19 23:10:29.671000

**Comment by elinz** [comment]
@jerrym26 Based on your screenshot, the publishing gp tool is throwing an invalid URL error. Can you verify that you can access your server in a web browser? Can you provide us the parameters you specified to run the PublishToOpsServer.py script? Thank you.

**Comment by jerrym26** [comment]
@elinz I was able to conect to my server and access it. The command line was PublishToOpsServer.py vm-def2.meudon.int 6443 admin Password Yes No Z:\G\OPSServerInstall\Install\Server\ServiceDefinitions

Thanks

**Comment by jerrym26** [comment]
@elinz I have made test this week-end on another installation  and I found that the problem came from a certificat security issue : during my first test I used Firefox in which I have accepted the exception raised by the certificat issue. But the ArcGIS for Server was still using the self-signed certificat issued by the application. When I tried to use internet Explorer I have found that the certificat error was present.
So I had the domain certificat to the application.
>> it works well

Thanks
